spec/ravelin/event_spec.rb

```diff
@@ -19,9 +19,9 @@ class MyDummyClass < Ravelin::RavelinObject
 
       it 'left as they are' do
         expect(event.serializable_hash).to eq({
-          'name' => 'John',
-          'timestamp' => 12345
-        })
+            'name' => 'John',
+            'timestamp' => 12345
+          })
```

Comment:
I think we need to stick with the Deliveroo formatting rules here (http://deliveroo.engineering/guidelines/ruby/#indentation-levels) would you mind squashing out this change and the one below?